src/pages/auth/verify-request.tsx

```diff
@@ -11,7 +11,7 @@ const VerifyRequest: NextPage = () => {
       <h1 className="mt-8 text-3xl font-bold">Check your email</h1>
 
       <div className="mt-4 text-xl">
-        A sign in link has been sent to your email address.
+        A link has been sent to your email address.
```

Comment:
How about `A verification link has been sent to your email address.`?

Reply:
True, that sounds better, implemented.

Reply:
How about something like `Please follow the instructions in the email we just sent you.`?

Reply:
Sounds perfect, I'll change it to that.